runhouse/resources/hardware/launcher_utils.py

```diff
@@ -155,8 +131,10 @@ def up(self, verbose: bool = True):
                     f"Received [{resp.status_code}] from Den POST '{self.LAUNCH_URL}': Failed to "
                     f"launch cluster: {load_resp_content(resp)}"
                 )
+            data = resp.json
 
         logger.info("Successfully launched cluster.")
+        return data
```

Comment:
whats the data that we're returning here? `cluster.up` should return a cluster object, so in the case of the den launcher should return `self.cluster`